### ISSUE TITLE
AutoComplete: NumpadEnter is not recognised

### ISSUE THREAD

**Issue by oselz** [primefaces/primevue#3816]
### Describe the bug

`<NumpadEnter>` is not recognised by AutoComplete component.

Autocomplete `onKeyDown` monitors `event.code` in which there is a difference between `<Enter>` and `<NumpadEnter>` and thus <NumpadEnter> cannot be used to select an auto completed item.

Instead `onKeyDown` should switch against `event.key` where both Enter keys will result in a `Enter` event key.

The issue is located here: https://github.com/primefaces/primevue/blob/8a6c3dfa9e3e9b16e1d43de1ad64e52faa8cf373/components/lib/autocomplete/AutoComplete.vue#L444

### Reproducer

https://codesandbox.io/p/sandbox/prod-snow-8msdfd

### PrimeVue version

3.26.1

### Vue version

3.x

### Language

ALL

### Build / Runtime

Vue CLI App

### Steps to reproduce the behavior

Type 'abc' into the autocomplete and press the `Enter` key adjacent the number pad on a full size keyboard.

### Expected behavior

Highlighted autocomplete item should be selected.

**Comment by oselz** [comment]
@tugcekucukoglu Appreciate the quick fix.

FYI, searching the codebase for `case: 'Enter':` shows a large number of other components have this identical issue.
